modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java

```diff
@@ -1025,6 +1027,137 @@ public void testScriptedAssignment() throws Exception {
 		_deactivateWorkflow(0, BlogsEntry.class.getName(), 0, 0);
 	}
 
+	@Test
+	public void testScriptedAssignmentNotifiableUsers() throws Exception {
+
+		// Roles Scripted Assignment
+
+		Organization organization = _createOrganization(true);
+
+		User organizationReviewerUser = _createUser(
+			_ORGANIZATION_CONTENT_REVIEWER, organization.getGroup());
+
+		_organizationLocalService.addUserOrganization(
+			organizationReviewerUser.getUserId(), organization);
+
+		User siteAdministratorUser = _createUser(
+			RoleConstants.SITE_ADMINISTRATOR);
+
+		_organizationLocalService.addUserOrganization(
+			siteAdministratorUser.getUserId(), organization);
+
+		_serviceContext = ServiceContextTestUtil.getServiceContext(
+			organization.getGroupId());
+
+		_activateWorkflow(
+			organization.getGroupId(), BlogsEntry.class.getName(), 0, 0,
+			_SCRIPTED_SINGLE_APPROVER_1, 1);
+
+		_addBlogsEntry(siteAdministratorUser);
+
+		WorkflowTask workflowTask = _getWorkflowTask(
+			organizationReviewerUser, null, false, null, 0);
+
+		Assert.assertEquals(
+			_sort(
+				Arrays.asList(
+					_adminUser, _companyAdminUser, organizationReviewerUser,
+					UserTestUtil.getAdminUser(_company.getCompanyId()))),
+			_sort(
+				_workflowTaskManager.getNotifiableUsers(
+					workflowTask.getWorkflowTaskId())));
+
+		_assignWorkflowTaskToUser(
+			organizationReviewerUser, organizationReviewerUser);
+
+		_completeWorkflowTask(organizationReviewerUser, Constants.APPROVE);
+
+		_deactivateWorkflow(
+			organization.getGroupId(), BlogsEntry.class.getName(), 0, 0);
+
+		_serviceContext = ServiceContextTestUtil.getServiceContext(
+			_group.getGroupId());
+
+		// User Scripted Assignment
+
+		_activateWorkflow(
+			0, BlogsEntry.class.getName(), 0, 0, _SCRIPTED_SINGLE_APPROVER_2,
+			1);
+
+		User user1 = UserTestUtil.addUser(
+			_company.getCompanyId(), _companyAdminUser.getUserId(),
+			StringPool.BLANK, "user1@liferay.com",
+			RandomTestUtil.randomString(
+				NumericStringRandomizerBumper.INSTANCE,
+				UniqueStringRandomizerBumper.INSTANCE),
+			LocaleUtil.getDefault(), RandomTestUtil.randomString(),
+			RandomTestUtil.randomString(), null,
+			ServiceContextTestUtil.getServiceContext());
+
+		_addBlogsEntry(user1);
+
+		workflowTask = _getWorkflowTask(user1, null, false, null, 0);
+
+		Assert.assertEquals(
+			Collections.singletonList(user1),
+			_sort(
+				_workflowTaskManager.getNotifiableUsers(
+					workflowTask.getWorkflowTaskId())));
+
+		_completeWorkflowTask(user1, Constants.APPROVE);
+
+		_deactivateWorkflow(0, BlogsEntry.class.getName(), 0, 0);
+
+		// Users Scripted Assignment
+
+		_activateWorkflow(
```

Comment:
Not needed https://github.com/seltong/liferay-portal/pull/367/files#diff-abad7071d64b0ba963ac1b66888f4ed40b1a53f9af37ad54ef93a3923a96b44cR1113-R1128